web-components/src/components/draggable/DraggableItem.ts

```diff
@@ -0,0 +1,36 @@
+import { CSSResultArray, html, LitElement, property } from "lit-element";
+import { customElementWithCheck } from "@/mixins/CustomElementCheck";
+import reset from "@/wc_scss/reset.scss";
+import styles from "./scss/module.scss";
+import { nothing } from "lit-html";
+
+export namespace DraggableItem {
+  @customElementWithCheck("md-draggable-item")
+  export class ELEMENT extends LitElement { 
+    @property({ type: Boolean }) row = false;
+
+    connectedCallback() {
+      super.connectedCallback();
+    }
+
+    static get styles(): CSSResultArray {
+      return [reset, styles];
+    }
+    
+    render() {
+      return html`
+        <div class="${this.row ? `md-draggable-row` : `md-draggable-item`}">
+          ${this.row ? html`<md-icon name="panel-control-dragger_16"></md-icon>` : nothing}
+          <slot></slot>
+        </div>
+      `;
+    }
+  }
+}
+
+
+declare global {
+  interface HTMLElementTagNameMap {
+    "md-draggable-item": DraggableItem.ELEMENT;
```

Comment:
You also need to add a new endpoint for the new component in the webpack file. Please read the updated CONTRIBUTING doc in web-component folder

Reply:
Done.